From leann.ogasawara@canonical.com  Wed Jun 16 13:29:49 2010
From: Leann Ogasawara <leann.ogasawara@canonical.com>
Date: Thu, 10 Jun 2010 15:49:24 -0700
Subject: Revert "USB: Adding support for HTC Smartphones to ipaq"
To: gregkh <gregkh@suse.de>
Cc: linux-usb <linux-usb@vger.kernel.org>
Message-ID: <1276210164.1221.3656.camel@emiko>


ipaq already had this device id defined:

{ USB_DEVICE(0x0BB4, 0x00CF) }, /* HTC USB Modem */

Revert the commit which adds the duplicate entry.

This reverts commit 04cab1329336d4577d6638360c905e360934b425.

Originally-by: Ben Collins <ben.collins@canonical.com>
Signed-off-by: Leann Ogasawara <leann.ogasawara@canonical.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ipaq.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/serial/ipaq.c
+++ b/drivers/usb/serial/ipaq.c
@@ -534,7 +534,6 @@ static struct usb_device_id ipaq_id_tabl
 	{ USB_DEVICE(0x413C, 0x4009) }, /* Dell Axim USB Sync */
 	{ USB_DEVICE(0x4505, 0x0010) }, /* Smartphone */
 	{ USB_DEVICE(0x5E04, 0xCE00) }, /* SAGEM Wireless Assistant */
-	{ USB_DEVICE(0x0BB4, 0x00CF) }, /* HTC smartphone modems */
 	{ }                             /* Terminating entry */
 };
 
